components/support/nimbus-fml/fixtures/fe/importing/coenrolling/app.yaml

```diff
@@ -0,0 +1,16 @@
+version: 1.0
```

Comment:
Nit: rename this to `app.fml.yaml`.

The naming scheme I was using was that `includes` use `.yaml`, and files that have a generated file associated with them (i.e. have an `about` block) use `.fml.yaml`.